libkbfs/errors.go

```diff
@@ -169,8 +169,12 @@ type ReadAccessError struct {
 
 // Error implements the error interface for ReadAccessError
 func (e ReadAccessError) Error() string {
+	t := tlf.Private
+	if e.Public {
+		t = tlf.Public
+	}
 	return fmt.Sprintf("%s does not have read access to directory %s",
-		e.User, buildCanonicalPathForTlfName(e.Public, e.Tlf))
+		e.User, buildCanonicalPathForTlfName(t, e.Tlf))
```

Comment:
Team folders will be in `/keybase/teams` or something like that.  I don't see what the issue is here -- it's passing the type (which could be `tlf.SingleTeam`) into `buildCanonicalPathForTlfName`, which picks the correct path type based on that.